### PR TITLE
Allow Rails 6.1

### PR DESCRIPTION
**Pull request by agrare**

### CODE REVIEW

**Review by kbrock — APPROVED** [review]
are we testing these changes to the ancillary gemspecs or just merging them?

**Comment by jrafanie** [comment]
> Just because this does deal with Active Record persistence so directly I'd like to run this through with cross-repo actually using rails 6.1, that way we can fix any issues in this PR

I'm currently trying this out with rails 6.1 locally with a manual cross repo run.

**Comment by agrare** [comment]
NOTE I've added rails 6.1 into the test matrix here

File: Gemfile [hunk 1]
```diff
@@ -14,4 +14,6 @@ when "5.2"
   gem "activerecord", "~>5.2.6"
 when "6.0"
   gem "activerecord", "~>6.0.4"
+when "6.1"
+  gem "activesupport", "~>6.1.4"
```
Comment (Fryguy) [inline Gemfile:18]:
```suggestion
  gem "activerecord", "~>6.1.4"
```

**Pull request merged**